### PR TITLE
[Documentation] Small fixes to "Run a FullNode"

### PR DESCRIPTION
**Pull request by JoshLind**
## Motivation

This PR makes some small fixes to the developer documentation: "Run a FullNode". Found here: https://aptos.dev/tutorials/run-a-fullnode

### Have you read the [Contributing Guidelines on pull requests](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md#pull-requests)?

Yes.

## Test Plan

Manually inspected all of the change sites.

## Related PRs

None.

### CODE REVIEW

File: developer-docs-site/docs/tutorials/run-a-fullnode.md [hunk 1]
```diff
@@ -81,11 +81,11 @@ During the initial synchronization of your FullNode, there may be a lot of data
   fullnode_1  | INFO 2020-09-28T23:16:04.508902Z execution/executor/src/lib.rs:580 sync_finished {"committed_with_ledger_info":false,"name":"chunk_executor","synced_to_version":634000}
   ```
 
-* At the same time, the StateSync component will output similar information but show the destination.
+* At the same time, the state sync component will output similar information.
 
-* The number of connections should be more than 0, `curl 127.0.0.1:9101/metrics 2> /dev/null | grep aptos_connections`. If it's not, see the next section for potential fix.
+* The number of connections should be more than 0, `curl 127.0.0.1:9101/metrics 2> /dev/null | grep aptos_connections`. If it's not, see the next section for potential solutions.
```
Comment (sherry-x) [inline developer-docs-site/docs/tutorials/run-a-fullnode.md:86]: maybe here we can specify that they should check the `outbound ` metrics, I think aptos_connections has both outbound and inbound, people might have inbound connection as 0

**Review by sherry-x — APPROVED** [review]

File: developer-docs-site/docs/tutorials/run-a-fullnode.md (hunk 1 above)
Comment (JoshLind) [reply developer-docs-site/docs/tutorials/run-a-fullnode.md:86]: Yeah -- makes sense! I'll follow up with some of my other fixes, too 😄 Thanks @sherry-x!

**Comment by radupopa369** [comment]
## Updated: Cachix Installation Workaround

If you encounter `error: public key is not valid` when installing cachix, use this workaround:

```bash
nix profile add nixpkgs#cachix --option trusted-public-keys "cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY="
```

This bypasses a key validation issue while still using the official NixOS cache.

---

**Why Determinate Nix?**

| Feature | Standard Nix | Determinate Nix |
|---------|-------------|-----------------|
| Flakes | Experimental | **Stable** |
| Parallel evaluation | No | **Yes (2x faster)** |
| Lazy trees | No | **Yes (3x faster, 20x less disk)** |
| License | LGPL v2.1 | LGPL v2.1 (same) |

Documentation has been updated in the latest commit.

**Pull request merged**